tensornetwork/ncon_interface_test.py

```diff
@@ -16,20 +16,39 @@
 from __future__ import print_function
 import pytest
 import numpy as np
+import tensornetwork as tn
```

Comment:
Prefer to not import `tensornetwork` directly in tests outside of the `tests/` directory. just use the modules directly like you do below.

Reply:
fixed